### PR TITLE
Fix url concatination logic for metron logic

### PR DESCRIPTION
**Pull request by Vitalii-Misechko**
* Functional Testing 
  * [x] Test cases
    * E2E tests on Draco LMS. No errors for main page and user drill page of Engagement Dashboard.
    
* Security, devops, perf: N/A
* UX and docs

  
FYI @anhill-D2L @hyehayes @johngwilkinson @NicholasHallman @bemailloux @MykolaGalian

### CODE REVIEW

**Review by johngwilkinson — APPROVED** [review]

File: model/dataApiClient.js [hunk 1]
```diff
@@ -12,12 +12,20 @@ export function restore() {
...
 const rolesEndpoint = '/d2l/api/ap/unstable/insights/data/roles';
 const semestersEndpoint = '/d2l/api/ap/unstable/insights/data/semesters';
-const dataEndpoint = '/unstable/insights/data/engagement';
+const dataEndpoint = 'unstable/insights/data/engagement';
 const relevantChildrenEndpoint = orgUnitId => `/d2l/api/ap/unstable/insights/data/orgunits/${orgUnitId}/children`;
 const ouSearchEndpoint = '/d2l/api/ap/unstable/insights/data/orgunits';
 const saveSettingsEndpoint = '/d2l/api/ap/unstable/insights/mysettings/engagement';
 const userDrillDataEndpoint = 'unstable/insights/data/userdrill';
 
+function concatMetronUlr(endpoint, apiPath) {
```
Comment (bemailloux) [inline model/dataApiClient.js:21]: function name has a typo in it :)

**Review by bemailloux — APPROVED** [review]

File: model/dataApiClient.js (hunk 1 above)
Comment (Vitalii-Misechko) [reply model/dataApiClient.js]: fixed that ))

**Comment by Vitalii-Misechko** [comment]
Should I create PR for release branch?

**Comment by Vitalii-Misechko** [comment]
> Should I create PR for release branch?

I should not the release branch uses LMS endpoint

**Comment by johngwilkinson** [comment]
> > Should I create PR for release branch?
> 
> I should not the release branch uses LMS endpoint

But probably for the other (tooltip and marker) PR.

**Pull request merged**